example_data/epilepsy_bundle.json

```diff
@@ -0,0 +1,656 @@
+                {
+                  "system": "http://loinc.org",
+                  "code": "48013-7",
+                  "display": "HGVS Description"
+                }
+              ]
+            },
+            "valueCodeableConcept": {
+              "coding": [
+                {
+                  "system": "http://www.ncbi.nlm.nih",
+                  "code": "gene_panel",
+                  "display": "NM_001271.3: c.3917del p.(Gln1306Argfs*13)"
+                }
+              ]
+            }
+          },
+          {
+            "code": {
+              "coding": [
+                {
+                  "system": "http://loinc.org",
+                  "code": "53034-5",
+                  "display": "zygosity"
+                }
+              ]
+            },
+            "valueCodeableConcept": {
+              "coding": [
+                {
+                  "system": "http://loinc.org",
+                  "code": "LA6706-1",
+                  "display": "Heterozygote"
+                }
+              ]
+            }
+          },
+          {
+            "code": {
+              "coding": [
+                {
+                  "system": "http://loinc.org",
+                  "code": "LL4034-6",
+                  "display": "Clinical significance"
+                }
+              ]
+            },
+            "valueCodeableConcept": {
+              "coding": [
+                {
+                  "system": "http://loinc.org",
+                  "code": "LA26332-9",
+                  "display": "Pathogenic"
+                }
+              ]
+            }
+          },
+          {
+            "code": {
+              "coding": [
+                {
+                  "system": "http://loinc.org",
+                  "code": "53034-0",
+                  "display": "inheritance"
+                }
+              ]
+            },
+            "valueCodeableConcept": {
+              "coding": [
+                {
+                  "system": "http://loinc.org",
+                  "code": "LA6705-0",
+                  "display": "AD"
+                }
+              ]
+            }
+          },
+          {
+            "code": {
+              "coding": [
+                {
+                  "system": "http://hl7.org/geneNames",
+                  "code": "gene-names",
+                  "display": "Gene Name"
+                }
+              ]
+            },
+            "valueString": "CHD2"
+          },
+          {
+            "code": {
+              "coding": [
+                {
+                  "system": "http://hl7.org/evidence",
+                  "code": "evidence",
+                  "display": "Evidence for classification of variant"
+                }
+              ]
+            },
+            "valueString": "absent from the gnomAD population database (PM2_Moderate). predicted null variant where loss of function is known disease mechanism (PVS1_very strong)."
+          },
```

Comment:
No. it doesn't. There is no other way to store this since each panel (variant details) has individual evidence. Some report has a panel while others could have two, three, or four panels with each having it's own evidence for the classification of variants.

Reply:
Yep agreed that each variant needs to have it's own set of evidence for classification. Might be useful to reference that the classification system here is the [ACMG/AMP guidelines](https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf). This seems like we should have a static page where we define our custom coding systems in the long term. Or is it worth putting this in `Observation.component.interpretation`?

Reply:
a further discussion needs to be had for how we define custom codes and where we have publically deployed pages definining their variables